CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
+## 2.1.0 (April 2, 2021)
+
+### ares-device
+* Supports screen capture using the `--capture-screen` option.
+
+### ares-generate
+* Updated the template of appinfo.json to apply new ACG groups.
+* Added '/' to config method in js_service templates.
+
+### ares-shell
+* Fixed an issue that environment variables using the `ares-shell -r` command were different from environment variables of the target device.
+
+## 2.0.3 (January 22, 2021)
+
+### ares
+* Updated help message.
+
+### Markdown documentation
```

Comment:
what about Documents?
The above meaning is too limited. I think it will need to be changed when pacakge.json is modified or another doc file is created.

Reply:
###Meta files